CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # CHANGELOG
 
+# Version 3.2.0-BETA.3
+* Add Spring integration (preview).
+* Add thread name to log capture. Resolves [#1699](https://github.com/microsoft/ApplicationInsights-Java/issues/1699)
+* Use azure monitor exporter instead of Open Telemetry exporter.
+* Add data truncation and sanitization to telemetry data.
+* Support W3C baggage propagation.
+* Update App Services internal diagnostics comments.
+* Disable legacy Request-Id propagation by default.
+* Handle partial content success due to Stamp specific redirects. Resolves [#1674](https://github.com/microsoft/ApplicationInsights-Java/issues/1674)
+* Log controlled warning when telemetry truncated. Resolves [#1021](https://github.com/microsoft/ApplicationInsights-Java/issues/1021)
+* Support system properties for the main "coordinates". Resolves [#1684](https://github.com/microsoft/ApplicationInsights-Java/issues/1684)
+* Small fixes to 2.x interop and exporter code cleanup.
+* Persist regular telemetries on disk inorder to retry when failed to send telemetry.
```

Comment:
this is not a change (well, I guess it's a change from BETA.1, but just back to normal)
```suggestion
```